packages/office-ui-fabric-react/src/components/Link/__snapshots__/Link.test.tsx.snap

```diff
@@ -6,10 +6,12 @@ exports[`Link renders Link correctly 1`] = `
       ms-Link
       {
         color: #0078d4;
-        outline: transparent;
-        position: relative;
         text-decoration: none;
       }
+<<<<<<< HEAD
```

Comment:
clean up these merge conflicts. You can simply rerun snaps and commit the new files.

Reply:
I'm getting a lot of weird errors rn when I run "npm run update-snapshots" so trying to fix them...